src/handlers/opportunitiesHandlers.ts

```diff
@@ -0,0 +1,134 @@
+import { getLogger } from '../logger';
+import { db } from '../database';
+import { isOpportunity, isOpportunityArraySchema } from '../types';
+import { ValidationError } from '../errors';
+import type {
+  Request,
+  Response,
+} from 'express';
+import type {
+  Opportunity,
+  CanonicalField,
+  ApplicationSchema,
+} from '../types';
+import type { ApplicationSchemaField } from '../types/ApplicationSchemaField';
+
+const logger = getLogger(__filename);
+
+const getOpportunities = (req: Request, res: Response): void => {
+  db.sql('opportunities.fetchAll')
+    .then((opportunitiesRaw) => {
+      logger.debug(opportunitiesRaw, 'opportunities.fetchAll');
+      type ShortOpportunity = Omit<Opportunity, 'applicationSchema'>;
+      type ShortApplicationSchema = Omit<ApplicationSchema, 'fields'>;
```

Comment:
Is there a better way than to make the type alias here? Maybe just repeat `Omit<ApplicationSchema, 'fields'>` several times?

Reply:
If you aren't re-using these names I don't think I'd use named types here (as opposed to just using the `Omit` in the relevant line).  Basically I think the name `Short___` is actually less meaningful than the raw `Omit` stuff.

That said I'm still in the process of grokking what's going on overall so maybe I'll have a better answer in a moment!

Reply:
Following up and seeing the lint error which is saying the same thing -- basically the best practice is this kind of type should not be named and should only be defined in context.

Here's a page that explains the rationale for the rule: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-type-alias.md

Reply:
I had to go back and look if I had used it more than once, and yes, I had used it twice below in the context, which was why I had created the alias. I read through https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-type-alias.md and maybe I didn't read it carefully enough but it did not seem to address the local definition of a utility type that would be repeatedly used within that local scope. Regardless, this question becomes moot (for now) when eliminating all the considerations of `ApplicationSchema` or `ApplicationForm` from `/opportunities`.